### PR TITLE
nanojava update (command line interface + Dockerization)

### PR DESCRIPTION
**Pull request by ammar257ammar**
Dear Egon,

I updated the nanojava library so the descriptors can be calculated for provided materials through a command line interface. The library is also now Dockerized so it can be used using docker.
The command line interface accepts formulas directly in the command or through an XML containing multiple materials. It is also able to write the calculated descriptors to a TSV file.
The docker image is now available through DockerHub where it can be used directly without compilation or docker build.

Please take a look, test it and accept the merge if you like it ^_^

Thank you.

Sincerely,
Ammar

### CODE REVIEW

File: src/main/java/org/bitbucket/nanojava/NanoJava.java [hunk 1]
```diff
@@ -0,0 +1,256 @@
+/* Copyright (C) 2011-2020  Egon Willighagen <egonw@users.sf.net>, Ammar Ammar <ammar257ammar@gmail.com>
```
Comment (egonw) [inline src/main/java/org/bitbucket/nanojava/NanoJava.java:1]: This is a new file and I did not contribute, so should not be listed as copyright owner.

Comment (egonw) [reply src/main/java/org/bitbucket/nanojava/NanoJava.java:1]: Oh, also, this file is started in 2020, so list only 2020.

File: src/main/java/org/bitbucket/nanojava/CliOptions.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
+/* Copyright (C) 2011-2020  Egon Willighagen <egonw@users.sf.net>, Ammar Ammar <ammar257ammar@gmail.com>
```
Comment (egonw) [inline src/main/java/org/bitbucket/nanojava/CliOptions.java:1]: This is a new file and I did not contribute, so should not be listed as copyright owner.

File: src/main/java/org/bitbucket/nanojava/io/Deserializer.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-/* Copyright (C) 2012-2013  Egon Willighagen <egonw@users.sf.net>
+/* Copyright (C) 2011-2020  Egon Willighagen <egonw@users.sf.net>, Ammar Ammar <ammar257ammar@gmail.com>
```
Comment (egonw) [inline src/main/java/org/bitbucket/nanojava/io/Deserializer.java:1]: Please change this to:

```java
/* Copyright (C) 2012-2013  Egon Willighagen <egonw@users.sf.net>
                      2020  Ammar Ammar <ammar257ammar@gmail.com>
```

File: src/main/java/org/bitbucket/nanojava/NanoJava.java (hunk 1 above)
Comment (ammar257ammar) [reply src/main/java/org/bitbucket/nanojava/NanoJava.java]: Done

File: src/main/java/org/bitbucket/nanojava/CliOptions.java (hunk 2 above)
Comment (ammar257ammar) [reply src/main/java/org/bitbucket/nanojava/CliOptions.java]: Done

File: src/main/java/org/bitbucket/nanojava/io/Deserializer.java (hunk 3 above)
Comment (ammar257ammar) [reply src/main/java/org/bitbucket/nanojava/io/Deserializer.java]: Done

**Comment by ammar257ammar** [comment]
Dear Egon,
I fixed the issues you raised in the comments.
Please check it and let me know if there is anything else.
Thank you.
Sincerely,
Ammar

**Pull request merged**